.opencode/skills/pytrendy/SKILL.md

```
@@ -0,0 +1,118 @@
+---
+name: pytrendy
+description: Use when modifying the trend detection pipeline, adding/renaming modules under pytrendy/, or working with datasets and results. Covers the 5-stage pipeline, module boundaries, built-in data, and the PyTrendyResults API.
+---
+
+# pytrendy package architecture
+
+## 5-stage pipeline
+
+`pytrendy/detect_trends.py` — the only entrypoint that matters. `detect_trends(df, date_col, value_col, plot=True, method_params=None, debug=False)` runs:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)